windows/tunnel/interfacewatcher.go

```diff
@@ -0,0 +1,156 @@
+/* SPDX-License-Identifier: MIT
+ *
+ * Copyright (C) 2019-2021 WireGuard LLC. All Rights Reserved.
+ */
+
```

Comment:
Is this code taken from the windows wireguard repo? Is this copyright header correct?

Reply:
Yes, all the code in this directory is taken from the wireguard-windows repository, which is under the MIT software license and I have simply removed a couple of functions. However, I forgot to include the copyright file that goes along with these files in order to be compliant with the MIT license.